src/windows/LottieReactNative/LottieAnimationViewManager.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+using Microsoft.ReactNative;
+using Microsoft.ReactNative.Managed;
+
+namespace LottieReactNative
+{
+    internal class LottieAnimationViewManager : AttributedViewManager<LottieAnimationView>
+    {
+        #region Constants
+
+        [ViewManagerExportedViewConstant()]
+        public int VERSION = 1;
+
+        #endregion
+
+        #region Creation
+
+        public override FrameworkElement CreateView()
+        {
+            var view = new LottieAnimationView();
+            view.HorizontalAlignment = HorizontalAlignment.Center;
+            view.VerticalAlignment = VerticalAlignment.Center;
+            view.PlaybackCompleted += View_PlaybackCompleted;
+            return view;
+        }
+
+        private void View_PlaybackCompleted(object sender, PlaybackCompleteEventArgs e)
+        {
+            if (sender is LottieAnimationView view)
+            {
+                ReactContext.UIDispatcher.Post(() => AnimationFinished?.Invoke(view, e.Canceled));
+            }
+        }
+
+        #endregion
+
+        #region Events
+
+        [ViewManagerExportedBubblingEventTypeConstant("animationFinish", "onAnimationFinish")]
+        public ViewManagerEvent<LottieAnimationView, bool> AnimationFinished;
+
+        #endregion
+
+        #region Properties
+
+        [ViewManagerProperty("sourceName")]
+        public void SetSourceName(LottieAnimationView view, string value)
+        {
+            view.SetAnimation(value);
+        }
+
+        [ViewManagerProperty("sourceJson")]
+        public void SetSourceJson(LottieAnimationView view, string value)
+        {
+            view.SetAnimationFromJson(value);
+        }
+
+        [ViewManagerProperty("resizeMode")]
+        public void SetResizeMode(LottieAnimationView view, string value)
+        {
+            view.SetResizeMode(value);
+        }
+
+        [ViewManagerProperty("progress")]
+        public void SetProgress(LottieAnimationView view, float value)
+        {
+            view.SetProgress(value);
+        }
+
+        [ViewManagerProperty("speed")]
+        public void SetSpeed(LottieAnimationView view, double value)
+        {
+            view.SetSpeed(value);
+        }
+
+        [ViewManagerProperty("loop")]
+        public void SetLoop(LottieAnimationView view, bool value)
+        {
+            view.SetLoop(value);
+        }
+
+        [ViewManagerProperty("imageAssetsFolder")]
+        public void SetImageAssetsFolder(LottieAnimationView view, string value)
+        {
+            view.SetImageAssetsFolder(value);
+        }
+
+        [ViewManagerProperty("colorFilters")]
+        public void SetColorFilters(LottieAnimationView view, IReadOnlyList<JSValue> value)
+        {
+            // TODO when we have an API to specify color filters

```

Comment:
We are working on the same thing :-D

I added some support for the color filters like that: https://github.com/mandrigin/lottie-react-native-1/commit/32d65c536e4493b94d5978dd361923d89339fdae

And before that support for the CodeGen files (required for that) like that: https://github.com/mandrigin/lottie-react-native-1/commit/8ced226eaba7367e177bac9eceb58989afd59301